extensions/adobe/experience/customerJourneyManagement/message-experienceevent.example.5.json

```diff
@@ -0,0 +1,45 @@
+{
+  "xdm:timestamp": "2020-06-15T10:10:00+00:00",
+  "@id": "https://data.adobe.io/experienceid/5ba0475a-5c31-4fce-b573-8ee222a40f45",
+  "xdm:eventType": "message.tracking",
+  "xdm:identityMap": {
+    "ECID": [
+      {
+        "xdm:id": "68519882713298129995549973016107434638",
+        "xdm:primary": true
+      }
+    ]
+  },
+  "xdm:messageExecution": {
+    "@id": "4218b775-bef3-46b2-aee2-7caae052cf94",
+    "xdm:messageID": "aa440b96-ce65-4ad2-b95d-8c3b51f89bc2",
+    "xdm:messageLabel": "Deal for Gold members",
```

Comment:
Do need message label?
Can labels on a message change overtime? any impact?

Reply:
I added this for the use cases where customer will be building the segmentation/reporting use cases in AEP/CJA. For usability purpose i added the label fields, otherwise it will be difficult for end user to build segments/reports using the machine generated ids.